app/models/verse.rb

```diff
@@ -2,4 +2,22 @@ class Verse < ApplicationRecord
   belongs_to :user
   belongs_to :competition
   has_many :votes
+
+  def self.last_month_winner
+    competition = Competition.find_by(month: previous_month)
+    vote_count = competition.verses.max do |verse|
+      verse.votes.count
+    end
+    vote_count
+  end
```

Comment:
If we ever make it to the stretch goal of multiple simultaneous competitions, we should be able to use this method with a `where` instead of a `find_by` for an easy transition